CMSgov/design-system
# [WNMGDS-751] Create react feature flags

## Summary
- Adds `flags.js` to `src/components` with new `ERROR_PLACEMENT_DEFAULT` flag
- Use new flags in form components, remove `Proptype` `defaultProps`
- Fix `react-docgen` error in child design systems without typescript option enabled

## How to test
- Setup and build the HC.gov child DS using the `bernard/feature-flag-test` branch, make sure to install from your local CMSDS branch
- Test usage of the HC.gov child DS form components in the CRA example and in a product app

## TODO:
- The doc site is still unfortunately not rendering the correct error placement, even when using the flag. Still unsure why.
